demos/benchmark/python/README.md

```diff
@@ -323,7 +325,6 @@ NO_PROXY=localhost no_proxy=localhost python3 /ovms_benchmark_client/main.py -a
           XI worker: --> dim: 300
           XI worker: Generated data shape: (1, 3, 300, 300)
           XI worker: start workload...
-...
```

Comment:
"..." means that there may be more output - it's not checked by tests so I don't think we need to remove it

Reply:
done